src/test/java/com/jfrog/ide/idea/scan/ScanBinaryExecutorTest.java

```diff
@@ -0,0 +1,90 @@
+package com.jfrog.ide.idea.scan;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.jfrog.ide.idea.scan.data.ScanConfig;
+import com.jfrog.ide.idea.scan.data.ScansConfig;
+import junit.framework.TestCase;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+
+/**
+ * @author tala
+ **/
+public class ScanBinaryExecutorTest extends TestCase {
+    private ScanBinaryExecutor scanner;
```

Comment:
I think that with this change, we can completely remove the setUp method
```suggestion
    private ScanBinaryExecutor scanner = new ApplicabilityScannerExecutor();
```